### DISCUSSION TITLE
tkinter / integrating image files

### DISCUSSION THREAD

**Discussion by zckin** [thonny/thonny#3099] [General]
Hi there,

I'm using Thonny to teach tkinter GUI design to my high school students who are already quite familiar with core concepts in CLI design and most fundamental coding concepts.

I'd like to use graphics (PNG, e.g.) in their tkinter projects, but I'm not sure how to do so.  I assume there's a reference to a filepath on a local machine OR a place to upload files into Thonny projects.  

Any assistance on this would be appreciated!  Full disclosure: we are using Macs.

**Comment by aivarannamaa** [comment]
I assume you want to find a way to refer to PNG files relative to the Python files. Here is an example how to do it: https://stackoverflow.com/a/1270970/261181
